WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/PrimaryButton.kt

```diff
@@ -32,6 +33,7 @@ fun PrimaryButton(
     isInProgress: Boolean = false,
     colors: ButtonColors = ButtonDefaults.buttonColors(
         contentColor = AppColor.White,
+        disabledContentColor = AppColor.White.copy(alpha = ContentAlpha.disabled),
```

Comment:
Yeah, in this scenario it's not ideal because the color should remain the same for both Light and Night modes, and it seems `contentColorFor` takes that into consideration, even though the background color (primary / jetpack_green_70) doesn't change, it chooses Black for Dark mode and White for Light mode. 🤷 

So it seems `contentColorFor` is somewhat the root cause for the issue and why I had to hardcode it to White both for Content Color and Disabled Content Color.

Reply:
Gotcha 👍🏻 :shipit: